contracts/contracts/sbtc-token.clar

```diff
@@ -1,5 +1,5 @@
-(define-constant err-not-token-owner (err u4)) ;; `tx-sender` or `contract-caller` tried to move a token it does not own.
-(define-constant err-not-protocol-caller (err u5)) ;; `tx-sender` or `contract-caller` is not the protocol caller
+(define-constant ERR_NOT_AUTH (err u600)) ;; `tx-sender` or `contract-caller` is not the protocol caller
+(define-constant ERR_NOT_OWNER (err u604)) ;; `tx-sender` or `contract-caller` tried to move a token it does not own.
```

Comment:
Thanks for updating this - I know I had semi conflicting comments on these errors, but this specific error we want to just be `u4` to stay in line with [SIP10's `transfer` function error codes](https://github.com/stacksgov/sips/blob/main/sips/sip-010/sip-010-fungible-token-standard.md#transfer)

Reply:
Updated!